### DISCUSSION TITLE
Uninstalling

### DISCUSSION THREAD

**Discussion by danreale** [abiosoft/colima#621] [Q&A]
Before uninstalling Colima, I had a bunch of containers and images still around that I forgot to remove. I stopped Colima and then uninstalled via brew(the way I originally installed). What happens to those containers and images? Are they still on my computer somewhere or are they deleted?

**Comment by abiosoft** [comment]
If you did not run `colima delete` before you uninstall, you can remove the `~/.colima` directory.

```
rm -rf ~/.colima
```

**Reply by danreale** [reply]
Thank you @abiosoft !
